Makefile

```diff
@@ -104,12 +104,15 @@ generate-py:  ## Generate .py files from notebooks
 .PHONY: generate-py
 
 
+AVATAR_BASE_URL ?= "http://localhost:8000"
+AVATAR_USERNAME ?= "user_integration"
+AVATAR_PASSWORD ?= "password_integration"
```

Comment:
From what I understand, here you're forcing these default for the whole file. Isn't it possible to put them in the `test-tutorial` recipe ?

Reply:
> Gonna be awesome combined with op cli :D

Oh, didn't even think about that!

Reply:
> From what I understand, here you're forcing these default for the whole file. Isn't it possible to put them in the `test-tutorial` recipe ?

These are Makefile variables, I can only define them in the scope of the whole file, so unfortunately no, I can't just add them to the `test-tutorial` recipe.
Though they have to be used with `${AVATAR_BASE_URL}` or `$(AVATAR_BASE_URL)`, and they are not automatically added to the environment. So they won't be used by accident.